src/index.css

```
@@ -139,43 +139,38 @@ html {
-	width: auto !important;
-	max-width: none !important;
-}
-
-div[class*='react-hot-toast'] > div {
-	white-space: nowrap !important;
-	min-width: fit-content !important;
+	max-width: min(calc(100vw - 32px), 520px) !important;
 	width: auto !important;
+	min-width: min(280px, 100%);
+	white-space: normal !important;
+	word-break: break-word;
```

Stylelint (17.12.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)